api/src/wfl/api/spec.clj

```diff
@@ -15,7 +16,9 @@
 (s/def ::workload-query (s/and (s/keys :opt-un [::all/uuid ::all/project])
                                #(not (and (:uuid %) (:project %)))))
 
-(s/def ::workflow-query (s/keys :opt-un [::all/status]))
+(s/def ::workflow-query (s/keys :opt-un [::all/status ::executor/submission]))
+
+(s/def ::retry-request (s/keys :opt-un [::all/status ::executor/submission]))
```

Comment:
All optional params per the spec, because we will enforce mandatory checks against a loaded workload.

Reply:
Happenstance, I could see down the road permitting workflow queries by additional filters that we don't support for retries, for ex.